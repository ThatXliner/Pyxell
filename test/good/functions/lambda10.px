
func f(Int n, Int->[Char] g) [Char] def
    return g(n)

a = [c for c in "abcde"]
print f(-2, a[_:])
print f(3, a[:_])
print f(-3, a[::_])
