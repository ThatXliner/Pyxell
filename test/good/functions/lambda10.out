d,e
a,b,c
e,b
